library/std/src/sys/pal/windows/c/windows_sys.rs

```diff
@@ -345,6 +345,10 @@ extern "system" {
     pub fn GetSystemTimeAsFileTime(lpsystemtimeasfiletime: *mut FILETIME) -> ();
 }
 #[link(name = "kernel32")]
+extern "system" {
```

Comment:
why are these individual extern blocks?

Reply:
Because that's what are binding generator generates. The [README](https://github.com/rust-lang/rust/blob/52f8aec14c616387c5f793687f2d9026de6c78ca/library/std/src/sys/pal/windows/c/README.md) has a few details about how this file is made.

Reply:
Basically it's all made by the [windows-bindgen](https://crates.io/crates/windows-bindgen) crate.

Reply:
ah